src/reynir_correct/errtokenizer.py

```diff
@@ -1789,6 +1847,65 @@ def suggest_word(
         token.add_corrected_meanings(m)
         return token
 
+    def add_ritmyndir_error(token: CorrectToken) -> CorrectToken:
+        """Add an error with corresponding correct value and details given info in Ritmyndir"""
+        # TODO At the moment the code assumes only one correct value is available in data
+        code = Ritmyndir.get_code(token.txt)
+        if code == "R001":
+            # Not an error
+            return token
+        if code == "SO-ÞGF4ÞF":
+            # Impersonal verbs that can have wrong subject case, not implicitly an error
+            return token
+        if code == "SN-TALA-GR":
+            # Unusual word forms, such as names with an article in the plural ('Jónarnir')
+            # Should not be marked as an error
+            return token
+        corrected = Ritmyndir.get_correct_form((token.txt))
+        if not corrected:
+            # No correct value available
+            return token
+        _, m = db.lookup_g(corrected, at_sentence_start=at_sentence_start)
+        text, details, refs = get_details(code, token.txt, corrected, m[0].stofn)
+        token.set_error(RitmyndirError(code, text, details, refs, token.txt, corrected))
+        if corrected not in {"in", "the", "for", "at"}:
```

Comment:
Convert this set to a frozenset() constant at the beginning of the file

Reply:
Done